src/main/java/seedu/address/model/person/Nric.java

```diff
@@ -56,7 +56,7 @@ public boolean equals(Object other) {
         }
 
         Nric otherNric = (Nric) other;
-        return nric.equals(otherNric.nric);
+        return nric.toLowerCase().equals(otherNric.nric.toLowerCase());
```

Comment:
does this require our tests to expand to check for non-capitalised/capitalised equivalence?